.env.example

```
@@ -52,6 +52,24 @@ ELEVENLABS_VOICE_SPEED=1.15
+# Redis -> env -> default in app/core/config/dynamic.py) so they're tunable at
+# runtime without a redeploy; the values here are the env-fallback defaults.
+# DRAGONTTS_URL is the proxy base. Templates opt into caching via
+# tts_configuration.enable_tts_caching=true (provider = the upstream, e.g.
+# cartesia); at call time they route through DragonTTS as model
+# "<provider>:<model>". A scheduler task probes DRAGONTTS_HEALTH_URL once per
+# BACKGROUND_TASKS_LOOP_INTERVAL_SECONDS (default 15s) and, after
+# DRAGONTTS_HEALTH_FAILURE_THRESHOLD consecutive failures, flips
+# enable_tts_caching to false on affected templates so calls fall back to the
+# upstream directly (it never rewrites provider/model). Set
+# ENABLE_DRAGONTTS_KILL_SWITCH=false to disable the monitor.
+DRAGONTTS_URL="http://dragontts.beta.svc.cluster.local"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)